### ISSUE TITLE
Possible BufferBuilders leak detected by ModernFix

### ISSUE THREAD

**Issue by Maypul** [RaphiMC/ImmediatelyFast#128]
1.19.2, forge, IF 1.26, alone or with 200 other mods, adding IF to modpack causes ModernFix (5.8.1) to print in log:
[15:55:49] [Finalizer/WARN] [ModernFix/]: One or more BufferBuilders have been leaked, ModernFix will attempt to correct this.

**Comment by RaphiMC** [comment]
Try reproducing this only with ImmediatelyFast and ModernFix and not 200 other mods

**Comment by Maypul** [comment]
Sorry if I was not clear, I meant it happens in both instances, ModernFix+IF and also ModernFix+IF+200 mods. Removing IF from both cases removes this warning

**Comment by RaphiMC** [comment]
I looked at the code and can confirm IF does indeed leak BufferBuilders. Its intentional tho and the leaked BufferBuilders are ones that Minecraft would have created and used (With IF they still get created, but never used and thus ModernFix classifies this as "leaked"). The problem with freeing them in IF: Other mods might expect their BufferBuilders to not get freed by ImmediatelyFast and would break if I free them. Freeing the vanilla allocated ones wouldn't save more than a couple of kilobytes, but could risk breaking other mods (Thats the reason why I removed the code that freed them). Here is the relevant code documenting this behaviour: https://github.com/RaphiMC/ImmediatelyFast/blob/6eda95cd4418c014e6c4672303178d40a2ce98b8/common/src/main/java/net/raphimc/immediatelyfast/injection/mixins/core/MixinVertexConsumerProvider.java#L48
